function_examples/random-word/README.md

```diff
@@ -1,3 +1,3 @@
-# Netlify Function Examples
-Basic Examples of using netlify functions
+# Random word
 
+This is a basic example of a function that when it gets called, returns a random word from an array of words back. This function has no build process and does not use any external packages.
```

Comment:
I would add that the function itself is stored in `/functions`, and the page with a link that calls the function is in `/_site`. I assume @futuregerald also made a demo site with this page, so I would link that so people can see how it works.